pkg/vdri/httpbinding/resolver.go

```diff
@@ -26,14 +30,21 @@ type didResolution struct {
 
 // resolveDID makes DID resolution via HTTP
 func (v *VDRI) resolveDID(uri string) ([]byte, error) {
-	resp, err := v.client.Get(uri)
+	req, err := http.NewRequest(http.MethodGet, uri, nil)
+	if err != nil {
+		return nil, fmt.Errorf("HTTP create get request failed: %w", err)
+	}
+
+	req.Header.Add("Accept", didLDJson)
+
+	resp, err := v.client.Do(req)
 	if err != nil {
 		return nil, fmt.Errorf("HTTP Get request failed: %w", err)
 	}
 
 	defer closeResponseBody(resp.Body)
 
-	if resp.StatusCode == http.StatusOK {
+	if resp.StatusCode == http.StatusOK && resp.Header.Get("Content-type") == "application/did+ld+json" {
```

Comment:
can we use the constant defined in line 21 here ? ie `didLDJson = "application/did+ld+json"`.